financial-templates-lib/logger/SlackTransport.js

```diff
@@ -60,6 +60,13 @@ const slackFormatter = info => {
           formattedResponse.blocks[
             formattedResponse.blocks.length - 1
           ].text.text += `    - _tx_: <https://etherscan.io/tx/${info[key][subKey]}|${info[key][subKey]}> \n`;
```

Comment:
FYI this string has a space at the end before `\n` but the string on line 69 does not have a space at the end.

Reply:
literally makes no difference but I'll remove it :)